Add IPv6 adoption documentation (#773)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_adopting-image-service-with-ceph-backend.adoc

the `openstack` namespace and that the `extraMounts` property of the
`OpenStackControlPlane` custom resource (CR) is configured properly. For more information, see xref:configuring-a-ceph-backend_migrating-databases[Configuring a Ceph back end].
+
[NOTE]

> If you are using IPv6, remember to change the load balancer IP to one that is correct in your environment. +
> E.g: `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`

Reviewer: ```suggestion
If you use IPv6, change the load balancer IP to the IP in your environment, for example,
 `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`.
```
